### ISSUE TITLE
Error: invalid pragma: exportpy, with hello world

### ISSUE THREAD

**Issue by flenniken** [yglukhov/nimpy#1]
I'm getting an error when trying out hello world.  I must have something setup wrong.  Here is what I did.

I installed it with nimble.

```
nimble install python
nimble list -i

docopt  [0.6.5]
python  [1.2]
```

I created a file called mymodule.nim with this:

```nim
import python
proc greet(name: string): string {.exportpy.} =
  return "Hello, " & name & "!"
```

I compiled the module with this:

    nim c --threads:on --tlsEmulation:off --app:lib --out:mymodule.so mymodule

I get the error:  Error: invalid pragma: exportpy

```
$ nim c --threads:on --tlsEmulation:off --app:lib --out:mymodule.so mymodule
Hint: used config file '/usr/local/Cellar/nim/0.17.2/nim/config/nim.cfg' [Conf]
Hint: system [Processing]
Hint: mymodule [Processing]
Hint: python [Processing]
Hint: dynlib [Processing]
Hint: strutils [Processing]
Hint: parseutils [Processing]
Hint: math [Processing]
Hint: algorithm [Processing]
mymodule.nim(2, 36) Error: invalid pragma: exportpy
```

**Comment by yglukhov** [comment]
Looks like you've installed a different python package https://github.com/nim-lang/python, which I did not know about... I guess I'll have to rename mine.

**Comment by yglukhov** [comment]
Should be fixed by 694938a. Please reopen if still an issue.
